docs/_includes/generated/netty_shuffle_environment_configuration.html

```diff
@@ -57,17 +57,23 @@
             <td>Number of extra network buffers to use for each outgoing/incoming gate (result partition/input gate). In credit-based flow control mode, this indicates how many floating credits are shared among all the input channels. The floating buffers are distributed based on backlog (real-time output buffers in the subpartition) feedback, and can help relieve back-pressure caused by unbalanced data distribution among the subpartitions. This value should be increased in case of higher round trip times between nodes and/or larger number of machines in the cluster.</td>
         </tr>
         <tr>
-            <td><h5>taskmanager.network.netty.client.connectTimeoutSec</h5></td>
-            <td style="word-wrap: break-word;">120</td>
+            <td><h5>taskmanager.network.retries</h5></td>
+            <td style="word-wrap: break-word;">0</td>
```

Comment:
This should be consistent with default value in `NettyShuffleEnvironmentOptions`. What is the motivation for changing this?

Reply:
It is a mistake.